src/Helper/GitHelper.php

```diff
@@ -727,11 +727,7 @@ public function getGitDir()
             return $this->topDir;
         }
 
-        try {
-            $this->topDir = $this->processHelper->runCommand(['git', 'rev-parse', '--show-toplevel']);
-        } catch (\RuntimeException $e) {
-            $this->topDir = '';
-        }
+        $this->topDir = $this->processHelper->runCommand(['git', 'rev-parse', '--show-toplevel']);
```

Comment:
Then, do you know why the `try/catch` block was removed? If the only allowed exception here is intended to be `fatal: Not a git repository (or any of the parent directories): .git`, I think we should check if the message matches and return `false` silently, throwing the exception otherwise. `GitHelperTest::returns_false_for_non_git_dir()` is failing because this change.